packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -41,16 +41,43 @@ import { error } from 'notification'
 const COLUMNS = [
   {
     name: _('vdiNameLabel'),
-    itemRenderer: vdi => (
-      <span>
-        <Text value={vdi.name_label} onChange={value => editVdi(vdi, { name_label: value })} />{' '}
-        {vdi.type === 'VDI-snapshot' && (
-          <span className='tag tag-info'>
-            <Icon icon='vm-snapshot' />
-          </span>
-        )}
-      </span>
-    ),
+    itemRenderer: (vdi, { baseCopiesByVdi }) => {
+      const baseCopy = baseCopiesByVdi[vdi.id]
```

Comment:
baseCopies

Reply:
Why should we put this variable in plural ?
`baseCopy` represente only one base copy.

Reply:
`baseCopiesByVdi[vdi.id]` can contain multiple base copies.

[See `_getBaseCopiesByVdi`](https://github.com/vatesfr/xen-orchestra/pull/5826/files#diff-724b302224d879069e88904114a1417b1e984950d8c202553cf0d387d1f754bdR388)